docs/getting_started/index.rst

```diff
@@ -86,21 +99,40 @@ Core-dev
     git clone https://github.com/<githubid>/project-pareto
     git clone git@github.com:<githubid>/project-pareto
 
-3. In this new project-pareto directory, run the following command which
-   installs PARETO in editable mode so that developers can make changes and
-   push to their fork/branch::
+3. Create a dedicated Conda environment for development work::
+
+    conda create --name pareto-dev python=3.9 --yes
+    conda activate pareto-dev
+
+4. Activate the ``pareto-dev`` Conda environment. This command must be run every time a new console/terminal window is opened::
 
-    pip install -e .
+    conda activate pareto-dev
+
+4. Navigate into the new ``project-pareto`` directory, then run the following command to install 
+   PARETO in editable mode and the development-only dependencies::
+
+    pip install -r requirements-dev.txt
+
+5. After installing PARETO, install the open-source solvers provided by the IDAES project::
+
+    idaes get-extensions --verbose
+
+6. (Recommended) install the pre-commit checks that will run automatically whenever ``git commit`` is used, preventing the commit from being created if any of the checks fail::
+
+    pre-commit install
+
+   .. note:: ``pre-commit`` can cause commits to fail for reasons unrelated to the pre-commit checks. For more information, check the `related GitHub issue(s) <https://github.com/project-pareto/project-pareto/issues?q=is%3Aissue+is%3Aopen+label%3Apre-commit>`_.
 
 .. _min_install_hybrid:
 
 Hybrid
--------
+------
+
 **User that can edit the base code**
 
-1. Create environment::
+1. Create and activate environment::
 
-    conda create -n pareto-env python=3.8 pip --yes
+    conda create -n pareto-env python=3.9 pip --yes
     conda activate pareto-env
 
 2. Download zip files (project-pareto-main)
```

Comment:
What zip files are we referring to in step 2 here? Can we include a hyperlink to the location to where a user can find the zip files?

Also, I think step 4 below could be made more clear. Specifically, I think we should mention that the command `pip install -r requirements-dev.txt` needs to be run from a terminal **_after_** navigating to the directory where the files are unzipped in step 3. I think this detail should probably be obvious to most people, but I think it's better to be explicit in the instructions.

Reply:
This should have been addressed by [5fbd7e7](https://github.com/project-pareto/project-pareto/pull/267/commits/5fbd7e7fd672a08eab891fa574ec96d1a874994e).

Reply:
@lbianchi-lbl Looks good to me, so I will resolve the thread, but I am curious - why have you included the link simply as text instead of as a proper hyperlink?